creusot-contracts/src/std/default.rs

```diff
@@ -1,94 +1,121 @@
+use crate as creusot_contracts;
+
 use creusot_contracts_proc::*;
-use std::default::Default;
 
-pub trait DefaultSpec: Default {
-    #[logic]
-    fn default_log() -> Self;
+pub trait Default: std::default::Default {
+    #[predicate]
+    fn is_default(self) -> bool;
+}
+
+extern_spec! {
+    mod std {
+        mod default {
+            trait Default where Self : Default {
+                #[ensures(result.is_default())]
+                fn default() -> Self;
+            }
+        }
+    }
+}
+
+// TODO: use a macro for these instances.
```

Comment:
What causes the group to appear, do you know?

Reply:
Can we have the macro generate a different type of group by inserting braces or parens?

Reply:
I guess that as soon as you use a meta-variable in a macro, it inserts a group. Here, we would like to use a meta-varaible for the default value, and it needs to appear in the pearlite code.

Reply:
How do the other macros that generate ord instances get around this then?